### DISCUSSION TITLE
Convert the website or create an PWA

### DISCUSSION THREAD

**Discussion by xandecelo** [JensRavens/thefetishtraveller#23] [Ideas]
@JensRavens 

I suggest to convert the website or create an PWA to make it "installable" on device and able to use resources like notifications without the need to use AppStore and early compliance with those guidelines.

Pinterest, Tinder are good examples: on iPhones you can "add to your home screen" and it becomes an application. Xbox Game pass is another example: visiting the site guides you to install it, with total session management.

I'm no Ruby guy, but I can handle  an implementation if we document the API and use OAuth for authentication.

**Comment by JensRavens** [comment]
Actually most of the work of this is already done. Manifests are written and the only missing thing is a service worker. 

Push will sadly be android only, so for that will write a wrapper based on Turbo iOS.

**Comment by JensRavens** [comment]
I'm happy to report that as of today, TFT is now a PWA and installable on android devices. Next up: push :)
